dnf-behave-tests/features/microdnf/install-xml-base.feature

```diff
@@ -0,0 +1,73 @@
+@no_installroot
+@desctructive
+Feature: microdnf install command on packages with xml:base
+
+@bz1734350
+Scenario: Install packages from local repodata with local packages
+#1. local repo with local packages
+Given I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "install setup"
```

Comment:
Since you're running the tests directly on the system and not in an installroot, there are the system repositories configured and they are being synced when you run microdnf, which takes a long time and may fail in CI due to networking not being available.

You should at least delete the repo configs like it is done e.g. here: https://github.com/rpm-software-management/ci-dnf-stack/blob/master/dnf-behave-tests/features/microdnf/repolist.feature

But, deleting the system repos when not running in an installroot should be part of the test environment setup somewhere in `environment.py`. If you'd like to try to figure out how to do that, that would be great. Actually, it should be pretty simple, when the test is tagged `@no_installroot`, just delete the system repos configuration, I think.